sources/europe/nl/PDOK-BGT-Basisregistratie-Grootschalige-Topografie

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "Feature",
+  "properties": {
+	"attribution": {
+           "required": true,
+           "text": "Kadaster, Basisregistratie Grootschalige Topografie (BGT)",
+           "url": "https://www.nationaalgeoregister.nl/geonetwork/srv/dut/catalog.search#/metadata/2891cc29-0a79-46d1-8649-287046d621c7"
+    },
+    "best": false,
+    "country_code": "NL",
+    "description": "Nationwide data set, Basic registration of large-scale topography, will be the detailed large-scale base map (digital map) of the whole of the Netherlands, in which the location of all physical objects such as buildings, roads, water, railway lines and (agricultural) land is registered in an unambiguous manner.",
+    "id": "BGT_standaardvisualisatie_WMTS",
+    "license_url": "https://community.openstreetmap.org/t/bgt-brt-akkoord-beschikbaarheid-voor-openstreetmap/80051",
+    "privacy_policy_url": "https://www.pdok.nl/privacy",
+    "max_zoom": 25,
+    "name": "PDOK BGT standaardvisualisatie",
+    "type": "tms",
+    "url": "https://service.pdok.nl/lv/bgt/wmts/v1_0?SERVICE=WMTS&REQUEST=GetTile&VERSION=1.0.0&LAYER=BGT standaardvisualisatie&STYLE=&FORMAT=image/png&tileMatrixSet=EPSG:3857&tileMatrix={zoom}&tileRow={y}&tileCol={x}",
```

Comment:
yes the link is not working for me aswell, @AllroadsNL i this can be fixed using "standaardvisualisatie
" instead of "BGT standaardvisualisatie" so the total correct url is:

https://service.pdok.nl/lv/bgt/wmts/v1_0?SERVICE=WMTS&REQUEST=GetTile&VERSION=1.0.0&LAYER=standaardvisualisatie&STYLE=&FORMAT=image/png&tileMatrixSet=EPSG:3857&tileMatrix={zoom}&tileRow={y}&tileCol={x}

Reply:
Yes, confirmed!

Reply:
Sorry for my mistake. I changed the layer=